cli/cmd/stat.go

```diff
@@ -307,6 +327,79 @@ func printStatTable(stats map[string]*row, resourceType string, w *tabwriter.Wri
 	}
 }
 
+func namespaceName(resourceType string, key string) (string, string) {
+	parts := strings.Split(key, "/")
+	namespace := parts[0]
+	namePrefix := getNamePrefix(resourceType)
+	name := namePrefix + parts[1]
+	return namespace, name
+}
+
+// Using pointers there where the value is NA and the corresponding json is null
+type jsonStats struct {
+	Namespace   string   `json:"namespace"`
+	Kind        string   `json:"kind"`
+	Name        string   `json:"name"`
+	Meshed      string   `json:"meshed"`
+	Success     *float64 `json:"success"`
+	Rps         *float64 `json:"rps"`
+	Latency_p50 *uint64  `json:"latency_p50"`
```

Comment:
agree re: your `latency_ms_p50` comment, i think it's best to include the unit somewhere.